.github/workflows/diff-change-to-dist.yaml

```diff
@@ -52,19 +52,30 @@ jobs:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
             const { readFile } = require('node:fs/promises')
-            const { comment } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
-
-            // Read diff from previous step
-            const diffText = await readFile('${{ github.workspace }}/dist.diff', 'utf8')
+            const { commentDiff } = await import('${{ github.workspace }}/.github/workflows/scripts/comments.mjs')
 
             // PR information
-            const number = '${{ github.event.pull_request.number }}'
+            const issueNumber = '${{ github.event.pull_request.number }}'
             const commit = '${{ github.event.pull_request.head.sha }}'
 
-            // Add or update comment on PR
-            await comment({ github, context }, number, 'Diff release', '## Changes to dist\n' +
-              '```diff\n' +
-              `${diffText}\n` +
-              '```\n\n' +
-              `SHA: ${commit}\n`
-            )
+            const diffs = [
+              {
+                path: '${{github.workspace}}/dist-js.diff',
```

Comment:
I know it's unlikely but I've developed an "oh no, Windows" reflex 😆

For some reason it only kicked in today so this isn't a blocker—but think we need `join()`?

Otherwise just double check all the spacing for consistency?

```suggestion
                path: '${{ github.workspace }}/dist-js.diff',
```

Reply:
The action is explicitely set to run on Ubuntu, so I don't think paths are a worry (I don't see why we'd want to broaden the diffing Github action to other platforms, unlike the build our contributors).

Regarding the spacing, how would you feel about setting us up to have Github workflow running scripts always depend on a proper JS file (say `<name-of-the-workflow>-<name-of-the-step>.js`), and just require the file and run it passing `github` and `context` + whatever other options are needed). This may make typing a bit wonky due to the type for `github` not being available, but will solve all formatting/linting issues 😊

Reply:
Yeah possibly, bit much for this one. Thanks for tidying it up